.scrutinizer.yml

```diff
@@ -0,0 +1,23 @@
+tools:
+    php_sim: false
+    php_pdepend: true
+    php_analyzer: true
+
+build:
+    environment:
+        php:
+            version: 5.6.0
+    tests:
+        override:
+            -
+                command: 'vendor/bin/phpunit --verbose --configuration phpunit.xml --coverage-clover test/clover.xml'
```

Comment:
It is: look [here](https://adayinthelifeof.nl/2013/11/20/external-code-coverage-with-travis-scrutinizer/)